automated install/basic-install.sh

```diff
@@ -375,40 +375,11 @@ package_manager_detect() {
     fi
 }
 
-select_rpm_php(){
-    # If the host OS is Fedora,
-    if grep -qiE 'fedora|fedberry' /etc/redhat-release; then
-        # all required packages should be available by default with the latest fedora release
-        : # continue
-    # or if host OS is CentOS,
-    elif grep -qiE 'centos|scientific|alma|rocky' /etc/redhat-release; then
-        # Pi-Hole currently supports CentOS 7+ with PHP7+
-        SUPPORTED_CENTOS_VERSION=7
-        SUPPORTED_CENTOS_PHP_VERSION=7
+check_epel_repo_required(){
+    # If the host OS is centos (or a derivative),
+    if grep -qiE 'centos|scientific|alma|rocky' /etc/redhat-release; then
```

Comment:
I would not perform a single positive check here... tmw another derivate pops up. Fake code:

```

if 'fedora|fedberry` then do nothing.
elif `centos` then install epel
else ask to install epal and inform about required packages (lighttpd\\n- lighttpd-fastcgi\\n- PHP version 7+)
```

Reply:
> else ask to install epal and inform about required packages (lighttpd\\n- lighttpd-fastcgi\\n- PHP version 7+)

My point in removing this part of the check is - we don't do this for any other OS. If someone wants to try installing on debberrybuntu 16, we allow them to at least try with `PIHOLE_SKIP_OS_CHECK`. But we do not put up any message s saying that you need to instal lighttpd, lighttpd-fastcgi etc etc

Without this check, the check for `fedora|fedberry` becomes redundant

Reply:
Ok. But then I would check 

`if [ ! fedora ] then` to catch all the (upcoming) derivated